VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        private static string ChangePassword = "PassWord917231";
```

Comment:
**CODE_SMELL**
> Add or update the header of this file.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWw_p9x51pMCJstp25Kb&pullRequest=17&resolved=false)

Reply:
/falsepositive

Reply:
The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWw_p9x51pMCJstp25Kb&pullRequest=17).